test/Mono.Linker.Tests.Cases/DataFlow/GetNestedTypeOnAllAnnotatedType.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+
+namespace Mono.Linker.Tests.Cases.DataFlow
+{
+	[RecognizedReflectionAccessPattern]
+	[SkipKeptItemsValidation]
```

Comment:
I didn't found any test that verifies the scenario when we want to keep something inside a nested type. Maybe we can have two nested classes and use the Kept item validation to verify that things are kept in one but not in the other

Reply:
Nevermind, I noticed that the GetNestedType does keep all the information inside the nested class